server/db/schema.sql

```diff
@@ -17,4 +19,23 @@ CREATE TABLE adoptable_pets(
     adoption_story TEXT NOT NULL,
     available_for_adoption BOOLEAN DEFAULT 't',
     pet_type_id INTEGER REFERENCES pet_types(id)
+);
+
+CREATE TABLE adoption_applications(
```

Comment:
this too looks like it should be in its own PR for Story #4!

Reply:
Story 4 is part of main